src/server.js

```diff
@@ -60,7 +64,7 @@ app.use((req, res) => res.sendStatus(404));
 // middleware error handler
 app.use((err, req, res, next) => {
   const defaultErr = {
-    log: { err: `An error occurred. ${err}` },
+    log: { err: `An error occurred. ${err.log ? err.log : err.message}` },
```

Comment:
Default error was still showing a log of Error object. This seems to resolve that and show our error log if it's an in house error, or shows the error message if it's an error from somewhere else.